fire/SFMainMod.F90

```diff
@@ -835,6 +1037,268 @@ subroutine  area_burnt_intensity ( currentSite, bc_in )
 
   end subroutine area_burnt_intensity
 
+  !*****************************************************************
+  subroutine  active_crown_fire ( currentSite)
+  !*****************************************************************
+
+    !evaluates if there will be an active crown fire based on canopy fuel and rate of spread
+    !returns final rate of spread and fire intensity in patch with added fuel from active crown fire.
+    !currentCohort%fraction_crown_burned is the proportion of crown affected by fire
+    ! TODO slevis: Jackie, you had ROS_torch coming in here but not used
```

Comment:
This far, the test
`SMS_Lm12_D_Mmpi-serial.1x1_brazil.I2000Clm50FatesCruRsGs.izumi_intel.clm-FatesFireLightningPopDens`
returns identical answers when I set
`fates_fire_active_crown_fire = 1`  in the params file, as when I set
`fates_fire_active_crown_fire = 0`

@jkshuman before I begin investigating (site too wet? no lightning? maybe it's time to switch to the site that @pollybuotte is running), I wanted to ask you about two variables:
- You had `ROS_torch` entering subroutine active_crown_fire but **not used**.
- You also had `canopy_bulk_denisty` calculated but **not used**.

Had you intended to use these variables? If so, where? If the answer is in S&R (2001), feel free to refer me there.

By the way, I stopped passing ROS_torch and four other variables as subroutine arguments, because my sense was that they wouldn't work as scalars and needed to be currentPatch% variables.

Reply:
@slevisconsulting canopy_bulk_density can be removed. it is no longer used.